utilities/src/Types/Hub.res

```diff
@@ -0,0 +1,48 @@
+type client
+type scope
+
+type rec hub<'t, 'a, 'b, 'c, 'd, 'e, 'f, 'g, 'i> = {
+  isOlderThan: float => bool,
+  bindClient: client => unit, //TODO Client.client
+  @deprecated("Use `withScope` instead.")
+  pushScope: unit => scope, //TODO  Scope.scope
+  @deprecated("Use `withScope` instead.")
+  popScope: unit => bool,
+  withScope: (scope => 't) => 't,
+  getClient: unit => Js.Nullable.t<client>,
+  getScope: unit => scope,
+  getIsolationScope: unit => scope,
+  captureException: (~exception_: 'b, ~hint: Event.eventHint<'a>=?, unit) => string,
```

Comment:
exception_ can this cause arg mismatch ?

Reply:
It wont bcuz in JS order of arguments matters, so the first argument is always exception and second is hint, but if the function captureException is defined with named param
like `({ param2: value2, param1: value1 })`
 then `exception` will go as `undefined` or `null` I guess, So im making this change for safer side